### ISSUE TITLE
commaSpace - do not lint comments 

### ISSUE THREAD

**Issue by charlierudolph** [SimenB/stylint#392]
Currently the commaSpace rule is being applied to comments
```styl
&:nth-of-type(2n+1) // matches 1,3,5,7 etc
```

**Comment by eddiemonge** [comment]
Have you tried 

```
&:nth-of-type(2n+1) /* matches 1,3,5,7 etc */
```

**Comment by eddiemonge** [comment]
ping @charlierudolph

**Comment by SimenB** [comment]
Both should work, help appreciated!

**Comment by PalleZingmark** [comment]
I can see that we [already strip out strings](https://github.com/SimenB/stylint/blob/master/src/checks/commaSpace.js#L26) so we don't get false positives with commas in strings. 
Should be possible to use the same approach with line comments.

Example: 
```javascript 
[...]

var removeQuotesRe = /(["'])(?:(?=(\\?))\2.)*?\1/g
var removeCommentRe = /\/\*[\s\S]*?\*\/|([^\\:]|^)\/\/.*$/gm

[...]

// just strip content between quotes and inline comments, leave rest of syntax intact
// this is so we don't get false positives with , in strings and inline comments
var trimmedLine = origLine.replace( removeQuotesRe, '""' ).trim()

if ( this.state.hasComment ) {
	trimmedLine = trimmedLine.replace( removeCommentRe, '$1' ).trim()
}

[...]
```
